docs/src/app/(docs)/api-reference/openapi-spec/scalar-client.tsx

```diff
@@ -56,8 +56,7 @@ export default function ScalarApiRef() {
         (a) =>
           a.type === "attributes" &&
           a.target instanceof Element &&
-          a.target.classList.contains("scalar-app") &&
-          a.target.classList.contains("references-layout"),
+          a.target.classList.contains("references-rendered"),
```

Comment:
cc @marclave can we have some `onLoaded` prop or `<Suspense>` support so we don't have to do this hacky mutation observer thing to unmount the loading state?

Reply:
sure!

Reply:
here you go      

https://github.com/scalar/scalar/pull/4750